### PR TITLE
Open conversations from the Dock menu

### PR DESCRIPTION
**Pull request by cawa-93**
image

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you mention it in the readme too? It's useful, but most apps don't have anything useful in the Dock menu, so users wouldn't think to look there.

**Comment by cawa-93** [comment]
@sindresorhus Would it be better to make such an image from the English system? Also, I think it would be worth adding the same functionality to other systems. However, I can not add and check it.

**Comment by sindresorhus** [comment]
@cawa-93 Sure, I can fill in the screenshot later if you add a short sentence about it in the readme.

**Comment by sindresorhus** [comment]
> Also, I think it would be worth adding the same functionality to other systems. However, I can not add and check it.

Windows has jump lists. We can open an issue after this for the Windows implementation with a "PR welcome" label.

**Comment by cawa-93** [comment]
@sindresorhus How about Linux? Gnome?

**Comment by sindresorhus** [comment]
I'm not very familiar with Linux, but I don't think it's possible there. If any Linux users know, they can do a pull request.

File: readme.md [hunk 1]
```diff
@@ -12,10 +12,17 @@ Caprine is an unofficial and privacy focused Facebook Messenger app with many us
...
-	<img src="https://c5.patreon.com/external/logo/become_a_patron_button@2x.png" width="160">
+	<img src="https://user-images.githubusercontent.com/1662812/40548807-66c154aa-603e-11e8-89b5-b514d426eafd.png" width="100">
 </a>
 
+### And Touch Bar support
+<img src="media/screenshot-touchbar.png" width="1085">
+
+<a href="https://www.patreon.com/sindresorhus">
+	<img src="https://c5.patreon.com/external/logo/become_a_patron_button@2x.png" width="160">
+</a>
```
Comment (sindresorhus) [inline readme.md]: These should be in the Highlights section, not here.

File: browser.js [hunk 2]
```diff
@@ -326,42 +326,29 @@ function closePreferences() {
...
-	const conversationListObserver = new MutationObserver(sendTouchBar);
-	conversationListObserver.observe(sidebar, {
-		subtree: true,
-		childList: true,
-		attributes: true,
-		attributeFilter: ['class']
-	});
+	ipc.send('conversations', conversations);
 }
+window.s = ipc.send;
```
Comment (sindresorhus) [inline browser.js]: ?

Comment (cawa-93) [reply browser.js]: Wow. Sorry. This is a piece of code I used for experimentation

File: readme.md [hunk 3]
```diff
@@ -12,10 +12,6 @@ Caprine is an unofficial and privacy focused Facebook Messenger app with many us
 	<img src="media/screenshot.png" width="846">
 </a>
 
-<a href="https://www.patreon.com/sindresorhus">
-	<img src="https://c5.patreon.com/external/logo/become_a_patron_button@2x.png" width="160">
-</a>
-
```
Comment (sindresorhus) [inline readme.md]: ?

**Comment by sindresorhus** [comment]
Looks good :)

**Pull request merged**